connection.js

```diff
@@ -843,7 +852,7 @@ function familyObjectMIBLIsminmax(familyNameString) { /* {{{ */
         ba = "above";
         lh = "high";
     }
-    const familyArray = ["", "ranked", "unranked"].map(e => familyNameString + e);
+    const familyArray = familyArrayFromFamilyNameString(familyNameString);
     return {argNameStrings: {all: familyArray[0], ranked: familyArray[1], unranked: familyArray[2]},
```

Comment:
Why not make `familyArrayFromFamilyNameString` return the  `{all: familyArray[0], ranked: familyArray[1], unranked: familyArray[2]}` object?

Reply:
Then you can do `return { argNameStrings: familyArrayFromFamilyNameString(familyNameString) };`

Reply:
```
function familyArrayFromFamilyNameString(familyNameString) {
    return {
        all: familyNameString,
        ranked: familyNameString + "ranked",
        uranked: familyNameString + "unranked"
      };
}
```